backend/danswer/server/models.py

```diff
@@ -102,8 +102,8 @@ class SearchResponse(BaseModel):
 
 
 class QAResponse(SearchResponse):
-    answer: str | None
-    quotes: dict[str, dict[str, str | int | None]] | None
+    answer: str | None  # DanswerAnswer
```

Comment:
Cant actually use the class due to circular import... there's like a few cross module dependencies that are very hard to remove